ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -238,7 +237,7 @@ public struct FluentNotification: View, TokenizedControlView {
                         image
                         textContainer
                         if !isFlexibleWidthToast {
-                            Spacer(minLength: horizontalSpacing)
+                            Spacer(minLength: 0)
```

Comment:
Is there a difference between `minLength: 0` and just not having any arguments?

Reply:
To my surprise, yes! Took me a bit to figure that out haha